c++/include/orc/Common.hh

```diff
@@ -122,6 +122,19 @@ namespace orc {
     StreamKind_BLOOM_FILTER_UTF8 = 8
   };
 
+  /**
+   * Specific read intention when selecting a certain TypeId.
+   * This enum currently only being utilized by LIST type selection.
+   *
+   * TODO: extend this to support MAP and UNION type.
```

Comment:
Apache ORC community recommends to use an IDed TODO with JIRA ID if needed.
```
- * TODO: extend this to support MAP and UNION type.
+ * TODO(ORC-XXX): extend this to support MAP and UNION type.
```

For this code, I'd like to recommend to remove this TODO and file an ORC JIRA instead.

Reply:
Removed the TODO. I will file a follow-up JIRA after this PR merged.